hello-vulkan/.google/packaging.yaml

```diff
@@ -0,0 +1,7 @@
+status:       PUBLISHED
+technologies: [Android, NDK, Vulkan, GameActivity]
```

Comment:
I'm not sure `GameActivity` falls under technologies, it might be an API instead.

Reply:
removed, thank you for reviewing!